roles/lib_utils/action_plugins/sanity_checks.py

```diff
@@ -54,6 +92,21 @@ def to_bool(var_to_check):
     return var_to_check in yes_list
 
 
+def check_for_removed_vars(hostvars, host):
+    """Fails if removed variables are found"""
+    found_removed = []
+    for item in REMOVED_VARIABLES:
+        if item in hostvars[host]:
+            found_removed.append(item)
+
+    if found_removed:
+        msg = "Found removed variables: "
+        for item in found_removed:
+            msg += "{} is replaced by {}; ".format(item[0], item[1])
```

Comment:
Can we make it clear that the replacement variable is the fully qualified reference to the image?

Reply:
I want to add a link to some documentation page with this error message.  We need a page that I can link to.

Reply:
Ok, we can follow up on that.